srcpkgs/mkinitcpio/files/lvm2_hook

```diff
@@ -1,24 +0,0 @@
-#!/usr/bin/ash
-
-run_earlyhook() {
-    mkdir /run/lvm
```

Comment:
I was not totally sure was is the purpose of this `mkir`. If it is not related to running `lvmetad`, it should be restored. The other parts are for `lvmetad` and can be dropped, else the hook will fail.